array-manipulation-3/src/unique.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-unused-vars */
+function unique(array) {
+  var unique = [];
+
+  if (array.length === 0) {
+    return unique;
+  } else if (unique.length === 0) {
+    unique.push(array[0]);
+  }
```

Comment:
at this point in the program for _sure_ `unique.length === 0`, so there's no need for `else if`